### ISSUE TITLE
Consider adding organization members / ZeroNet repo maintainers

### ISSUE THREAD

**Issue by DaniellMesquita** [HelloZeroNet/ZeroNet#1652]
@HelloZeroNet @shortcutme 
The project is totally centralized on you.
If you isn't here anymore, ZeroNet will have a fork?

I recommend those maintainers:
@krixano @imachug @tangdou1

**Comment by DaniellMesquita** [comment]
I would also recommend @ZeroMux, but he seem to be away.

**Comment by imachug** [comment]
Even if some of us becomes a maintainer, we should use Pull Requests, so we *can* directly commit to the repository, but only if that's *very* important -- e.g. if there's a security vulnerability. Otherwise, we should use PRs, so we and @HelloZeroNet could check each other's code.

**Comment by imachug** [comment]
Currently @HelloZeroNet is a user not an organization. AFAIK GitHub and GitLab have free organizations. On GitHub, you have to set your billing email, however, AFAIK they don't send anything if your plan is `Free`. However, **the owners of the organization can change the plan, so make sure you don't give the owner role to people you don't know well** (er... maybe including me).

**Comment by DaniellMesquita** [comment]
@HelloZeroNet can rename its user name, then create a organization with same name.

**Comment by DaniellMesquita** [comment]
@PeterSurda
Would you be a good maintainer?

**Comment by PeterSurda** [comment]
@DaniellMesquita no, I don't have any particular interest in ZeroNet (or time for it). I just read about #65 in an interview, so I spent a couple of hours writing it and that's about it.

**Comment by DaniellMesquita** [comment]
@HelloZeroNet @shortcutme
